tests/cases/fourslash/quickInfoEvolvingArrayType.ts

```diff
@@ -0,0 +1,15 @@
+///<reference path="fourslash.ts" />
+
+// @strict: true
+// @checkJs: true
+// @allowJs: true
+// @outDir: ./dist
+
+// filename: index.js
+//// var C = [];
+//// for (var a in C)

```

Comment:
Also, my bad, this should probably have an opening `{`, the trailing `}` is unmatched.

Reply:
I figured that was part of the test, hah.